### ISSUE TITLE
Notification "Archives of type text/html are not supported" while enabling app in 8.1.3

### ISSUE THREAD

**Issue by tuneil** [DjazzLab/ocdownloader#37]
Hi,
Got an issue for the fresh-installed OC 8.1.3, after activation see next message:

Archives of type text/html are not supported
and application is not activated.

By looking at this topic, wonder if we have the same issue as developer of ownnote recently got
Fmstrat/ownnote#174

Thanks,
Andrii

**Comment by DjazzLab** [comment]
Hi, thanks for the link to the ownote page.
I'll have time to analyze this in the next days...

**Comment by tuneil** [comment]
Just an update on this(could be useful for others), workaround seams pretty simple - manually install application by downloading and unpacking archive with app files.

**Comment by malevolent** [comment]
Hi there!

I had also the same issue than the OP, and I just downloaded the ZIP from here and uncompress under owncloud/apps but I cannot enable it anywhere... I can only see the ocDownloader NG from the store and it says the same?

How did you achieved the installation?

**Comment by DjazzLab** [comment]
Hi ! Did you rename the application folder from "ocdownloader_master" to "ocdownloader" ?

**Comment by malevolent** [comment]
Yes! and applied permissions to apache user (http) and let the permissions to 755

**Comment by tuneil** [comment]
malevolent, please check the section "Not Enabled" on the Apps page, your app should be there.

**Comment by malevolent** [comment]
Thank you tuneil!!!

 I would have sworn I checked all the sections... but obviously I didn't because now it's installed.
